server/apps/archive/archive_link.py

```diff
@@ -59,3 +57,16 @@ def create(self, docs, **kwargs):
         doc.update(linked_item)
         build_custom_hateoas(CUSTOM_HATEOAS, doc)
         return [linked_item['_id']]
+
+    def _validate_link(self, target, target_id):
+        """
+        Validates the article to be linked
+        :param target: article to be linked
+        :param target_id: id of the article to be linked
+        :raises: SuperdeskApiError
```

Comment:
Can we elaborate this doc bit more? 

`raises` should list all the possible exceptions the method is going to throw